lib/Gateway.js

```diff
@@ -303,6 +300,10 @@ function onValueChanged (valueId, node, changed) {
       data = { time: Date.now(), value: tmpVal }
   }
 
+  if (typeof data === 'object' && this.config.includeNodeInfo) {
```

Comment:
```suggestion
  if (this.config.includeNodeInfo && typeof data === 'object') {
```

This order makes the check quicker